Dockerfile

```diff
@@ -5,8 +5,8 @@ WORKDIR /app
 RUN apt update && apt -yq install libpq-dev
 # install manually to workaround issue in psycopg2-binary 2.9.5
 RUN pip3 install psycopg2-binary --no-binary psycopg2-binary
-COPY requirements-dev.txt requirements.txt
-RUN python3 -m pip install --upgrade pip && pip install -r requirements.txt
+COPY requirements-dev.txt requirements-dev.txt
+RUN python3 -m pip install --upgrade pip && pip install -r requirements-dev.txt
```

Comment:
Same question as here: https://github.com/communitiesuk/funding-service-design-assessment/pull/242#discussion_r1190829767

Reply:
Look for the reply https://github.com/communitiesuk/funding-service-design-assessment/pull/242#discussion_r1190841164